astropy/visualization/units.py

```diff
@@ -65,7 +65,7 @@ def axisinfo(unit, axis):
             elif unit == u.degree:
                 return units.AxisInfo(
                     majloc=ticker.AutoLocator(),
-                    majfmt=ticker.FormatStrFormatter("%i°"),
+                    majfmt=ticker.FormatStrFormatter("%g°"),
```

Comment:
Assuming that the idea here was just to have the degree symbol, this definitely is the right change (we'll leave $1\fdg5$ for another time...)

Reply:
(what's `\fdg` ?)

Reply:
An aastex thing that puts the degree symbol over the decimal point.